docs/Imports.md

```diff
@@ -12,13 +12,18 @@ There are two types of resources that are supported in imports: *http(s)* and *f
 import "http://mywdlrepository/my.wdl" as http_import1
 import "https://github.com/broadinstitute/cromwell/blob/master/engine/src/main/resources/3step.wdl" as http_import2
 ```
-To use a file-based import resource, you must provide a ZIP bundle of your resources and then use a path relative to that ZIP in your import statement. For example:
+To use a file-based import resource, provide a ZIP bundle of your resources and then use a path relative to that ZIP in your import statement. For example:
 
 ```wdl
 import "my-wdl-in-the-root-directory.wdl" as file_import1
 import "my/wdl/sub/directory/example.wdl" as file_import2
 ```
 
+If there exists a `my/wdl/sub/directory/imports/importing_an_import.wdl`, `my/wdl/sub/directory/example.wdl` could import it relatively like so:
```

Comment:
hmm, I found this a little bit hard to understand. I might provide a bit of context, eg 

"imports from your submitted workflow are evaluated relative to the base of the zip file. In other case, import paths are relative to the file you are currently importing from, for example <<your example>>"